tests/html_attributes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -296,11 +403,41 @@ width="205px" alt="Yahoo"/></a></h1></div></div>
 =====================================output=====================================
 <img class="foo bar" />
 
-<img class="  " />
+<img class="" />
 <img class />
 
 <img
-  class="looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong a-long-long-long-long-long-class-name another-long-long-long-class-name foo bar foo bar foo bar foo bar foo bar foo bar foo bar foo bar foo bar foo bar foo bar foo bar foo bar"
+  class="
+    looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong
+    a-long-long-long-long-long-class-name
+    another-long-long-long-class-name
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
+    foo
+    bar
```

Comment:
Yes, this looks not exactly good, but this is not a real-life example.

Reply:
https://deploy-preview-7865--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEACdqA8ATAlgN1QEMAnXIgWgAtdtsEBeAHRBhIFc4XUwAbIgM4DmIAEYQS9EqhIR2UetgoBGVADNecAB4UBVMlADWFAAypRAcwoWSRAJ6oADjtXwtMa7bsVeuC1RhUAFteFXUANlQlKGg4KIpoCigifAo1CDB2AVQAKwEKHPYgxwoYCApRImwLOAKixxYAPiYoDAwAKXrUMpa29EwBRyJWvkFhFiVcKF8oWtFeDMNg0NVUol8LKAog2mxNJsBWwkwAekHh5taME7x8RpAAGhAIRxhcaAFkUFJZAHcABVICA+KHWP3sH0eolsYEMcBgAGUhmAphZkGxOI8AiEAOo0eBnMBweFA3CvfCkuzIcBCB4gKYCOAkGB-WwWIJEZBqdYMx55LQAIWhsIRRCCcAAMlM4JzuXBeQItPCUZoAIrsCDwGW8HkgIYkBkkKlY3i0xwGGDY2gwKjIAAcJkeZogDOxtkcVLNcAN+GljwAjur4CznsCQIIknA4IpaSQ4AHcLGWUQ2RykFztXKQAydmiOJmBMq4GqNdK07LHjAiKJLdhrcgAEwV2y4GYWADCECC7KpXoArLSsnAACpV4HpnX4TgASQUCARYDILwAggp4TA7JotQyAL7boA

Code from this github PR page https://github.com/prettier/prettier/pull/7865

Reply:
Seems like there is no good way to format this real example. Probably we should simply retain new lines in the `class` attribute, only re-indenting them if needed. That would be pity, the idea with prefixes looked good.

Reply:
(from my perspective at least, the output in @fisker 's playground is great)